users/test_user_upsertion.py

```diff
@@ -50,12 +49,38 @@ def test_insert_valid_user(self):
         # original user attributes matches found user attributes
         self.assertEqual(user_to_insert["user_id"], found_user["user_id"])
         self.assertEqual(user_to_insert["name"], found_user["name"])
-        self.assertEqual(
-            user_to_insert["is_organizer"], found_user["is_organizer"])
```

Comment:
To make sure the field is set correctly, you should probably still check that `found_user["is_organizer"]` is `False` here.

Reply:
Done https://github.com/knative-portability/large-events/pull/33/commits/775384ce54f94cc2e9a0fb16c2c65b0c302de55a